src/scenes/data-scene/data-scene-config.js

```diff
@@ -12,8 +14,10 @@ export default {
   globe: {
     activeLayers: [
       { title: GRAPHIC_LAYER },
+      { title: CITIES_LABELS_LAYER },
```

Comment:
So we are always adding these layers from the beginning. Don't we need to add them in other scene configs? I actually think this could be improved in the future and style the layers in some other place. Maybe as these are ISO dependant will be a bit more difficult. But its weird to me that this component just styles these layers and is in the middle of the sceen.